pkg/secrets/pushtofile/secret_spec.go

```diff
@@ -98,6 +117,25 @@ func validateSecretPaths(secretSpecs []SecretSpec, groupName string) []error {
 	return errors
 }
 
+func validateSecretContents(secretSpecs []SecretSpec, groupName string) []error {
+	var errors []error
+	for _, secretSpec := range secretSpecs {
+		if err := validateSecretContent(secretSpec.ContentType, groupName); err != nil {
+			errors = append(errors, err)
+		}
+	}
+	return errors
+}
+
+func validateSecretContent(content, groupName string) error {
+	if content == "text" || content == "base64" {
+		return nil
+	} else {
+		return fmt.Errorf("secret group %s: the content-type of %s is invalid",
```

Comment:
Somewhat similar to the other comment - in the K8s secrets implementation, if a user provides an invalid content-type we log a warning and assume it should be treated as `text` so that it doesn't fail

Happy to discuss which outcome is more intuitive for the user, but either way we should aim to be consistent